bin/lb-ng.js

```diff
@@ -1,26 +1,31 @@
 #!/usr/bin/env node
 
-var fs = require('fs');
 var path = require('path');
+var SG = require('strong-globalize');
+SG.SetRootDir(path.resolve(__dirname, '..'));
```

Comment:
You're the first one to run into this case, but I see your point. Do you need to SetRootDir outside index.js though?